packages/playwright/src/runner/runner.ts

```diff
@@ -134,7 +134,10 @@ export class Runner {
   async watchAllTests(): Promise<FullResult['status']> {
```

Comment:
Let's get rid of this method entirely, inline it into `program.ts` and not create `Runner` for watch mode at all.

Reply:
done!